api/src/main/java/net/starype/quiz/api/game/Question.java

```diff
@@ -0,0 +1,36 @@
+package net.starype.quiz.api.game;
+
+import net.starype.quiz.api.game.player.UUIDHolder;
+
+import java.util.List;
+
+public interface Question extends UUIDHolder {
+
+    List<QuestionTag> getTags();
+
+    default boolean isTagAttached(QuestionTag tag) {
+        return getTags().contains(tag);
+    }
+
+    void registerTag(QuestionTag tag);
+
+    void forgetTag(QuestionTag tag);
+
+    QuestionDifficulty getDifficulty();
+
+    String getQuestion();
+
+    String getCorrectAnswer();
+
+    int getMaxTrialAmount();
+
+    boolean trySubmitAnswer(UUIDHolder entity, String answer);
```

Comment:
This should return a double between 0 and 1, representing the correctness of the answer